source/fe/fe_dgq.cc

```diff
@@ -656,10 +657,17 @@ FE_DGQ<dim, spacedim>::memory_consumption () const
 
 
 
+// ------------------------------ FE_DGQArbitraryNodes -----------------------
+
 template <int dim, int spacedim>
 FE_DGQArbitraryNodes<dim,spacedim>::FE_DGQArbitraryNodes (const Quadrature<1> &points)
-  : FE_DGQ<dim,spacedim>(points)
-{}
+  : FE_DGQ<dim,spacedim>(Polynomials::generate_complete_Lagrange_basis(points.get_points()))
```

Comment:
I could have kept the old constructor as well because it is really only 10 lines of code, but I think that this one is more general. We should still document this. I will add a not to the list of incompatible changes and show the workaround. Or do you want me to deprecate the (protected) constructor instead? I guess the use case is quite narrow so we might save the deprecation here...